scripts/mina-sync-monitor.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+exit_code=1
+
+check_sync_status() {
+  status=$(docker exec mina mina client status --json | jq -r .sync_status || echo "Null")
+  echo "Current sync status: $status"
+}
+max_attempts=24
+attempt=0
+sleep_duration=300
```

Comment:
**nitpick:** We could move the variables initialization to the top, and also add `status="Null"`.